package.json

```diff
@@ -63,7 +63,11 @@
     "travis-after-all": "1.4.4",
     "webpack": "2.2.1",
     "webpack-combine-loaders": "2.0.3",
+    "webpack-dashboard": "^0.3.0",
     "webpack-dev-server": "2.3.0",
     "xhr": "2.3.3"
+  },
+  "dependencies": {
+    "postcss-modules-values": "^1.2.2"
```

Comment:
This should be included in `devDependencies`. Please use absolute version numbers for 3rd party dependencies.